### PR TITLE
Increase API version

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Depends on https://github.com/vmware/go-vcloud-director/pull/300

* Fix test after API changes in govcd

### CODE REVIEW

File: vcd/resource_vcd_edgegateway_test.go [hunk 1]
```diff
@@ -729,9 +729,8 @@ resource "vcd_edgegateway" "egw" {
 	configuration           = "compact"
 	advanced                = true
   
-	# FIPS mode is not set because otherwise provider would try to send this field
-	# and vCD 9.0 complains about it, because it was introduced in vCD 9.1 only
-	# fips_mode_enabled               = false
+    # can be only true when system setting Allow FIPS Mode is enabled
+	fips_mode_enabled               = false
```
Comment (Didainius) [inline vcd/resource_vcd_edgegateway_test.go:733]: Great fix - at last :)
Can I ask to also do a few cleanups based on this before we run into the testing parade.
* Remove that comment and description https://github.com/terraform-providers/terraform-provider-vcd/blob/v2.7.0/vcd/resource_vcd_edgegateway.go#L250-L252
* The extra "note" for 9.0 is not needed anymore in https://www.terraform.io/docs/providers/vcd/r/edgegateway.html#fips_mode_enabled
* Remove the warning https://github.com/terraform-providers/terraform-provider-vcd/blob/v2.7.0/vcd/resource_vcd_edgegateway.go#L355-L358

Comment (vbauzysvmware) [reply vcd/resource_vcd_edgegateway_test.go:733]: Added changelog

Comment (vbauzysvmware) [reply vcd/resource_vcd_edgegateway_test.go:733]: Well, regarding cleanup. My idea was at this moment just increase version if everything is fine - we make general cleanup, I created Explicit task for next version.

Comment (Didainius) [reply vcd/resource_vcd_edgegateway_test.go:733]: Ok, let it be then

Comment (lvirbalas) [reply vcd/resource_vcd_edgegateway_test.go:733]: @vbauzysvmware Is there a good reason not to do the cleanup now in these PRs? As we decided to release a focused 2.8.0 just to support 10.1 we have time and I believe we should do the cleanup now, so it's not lost in the technical debt bucket.

Comment (vbauzysvmware) [reply vcd/resource_vcd_edgegateway_test.go:733]: Yes main reason is to split testing between API versioning changes and technical debt. We decide when todo cleanup in same version or next. I created task to track this.

**Pull request merged**